build/stf-run-ci/defaults/main.yml

```diff
@@ -45,6 +45,7 @@ prometheus_webhook_snmp_image_tag: latest
 sgo_bundle_image_tag: latest
 sto_bundle_image_tag: latest
 stf_index_image_tag: latest
+# Why is this v1? We can't create the bundle with v1, need v0.19.4
```

Comment:
I think you already found the answer to this, but for the record it's because the `operator-sdk run bundle` command that we use for testing installable OLM bundles is only availabe in > v1